Tasks/Common/utility-common/module.json

```diff
@@ -0,0 +1,5 @@
+{
+  "messages": {
+    "TokenUnavailable": "Could not retrieve token credentials."
```

Comment:
Can you clarify that the token credentials spoken of are for "TFS/Team Services packaging"?